pygeoapi/provider/elasticsearch_.py

```diff
@@ -336,13 +307,15 @@ def query(self, offset=0, limit=10, resulttype='results',
                             next(gen)
                     except StopIteration:
                         break
-                results['hits']['total'] = \
-                    len(results['hits']['hits']) + offset
+
+                matched = len(results['hits']['hits']) + offset
+                returned = len(results['hits']['hits'])
             else:
-                results = self.es.search(index=self.index_name,
-                                         from_=offset, size=limit,
-                                         body=query)
-                results['hits']['total'] = results['hits']['total']['value']
+                es_results = self.es.search(index=self.index_name,
+                                            from_=offset, size=limit, **query)
+                results = es_results.body
```

Comment:
The es_results structure no longer has a 'body' element:
`results = es_results.body` -> `results = es_results`